Yoast/wordpress-seo
# Fix RTL for DraftJS

## Summary

This PR can be summarized in the following changelog entry:

* Fixes text directionality for the title and description fields in the snippet editor for right-to-left languages.

## Relevant technical choices:

* We were inverting the DraftJS CSS, but that CSS already has measures for
RTL and LTR. So we need to not invert the CSS.

## Test instructions

This PR can be tested by following these steps:

* Link yoast-component's `stories/add-RTL-theme-to-snippet-editor`
* Set your WordPress site to a right-to-left language.
* See that the replacement variables are aligned on the right side of the title and description fields.
* Test whether nothing breaks for LTR languages.

## Quality assurance


Fixes #10208
